src/main/java/org/apache/commons/io/IOUtils.java

```diff
@@ -341,6 +341,7 @@ public static byte[] byteArray() {
 
     /**
      * Returns a new byte array of the given size.
+     * Throws java.lang.NegativeArraySizeException if the size is negative.
```

Comment:
This is in the wrong place, use a throws tag.

Reply:
Hi garydgregory,
   I fixed it with @throws, you can review it again.